docs/source/release_notes.rst

```diff
@@ -33,49 +59,78 @@ Minor Changes
 
-- ``zos_job_query`` - Fixed an issue that when using a job ID with less than 8 characters would result in a traceback. The fix
+- ``zos_job_query`` - Fixed an issue that when using a job ID with less than 8 characters, would result in a traceback. The fix
   supports shorter job IDs as well as the use of wildcards.
 
 - ``zos_unarchive``
 
-    - Fixed an issue when using a local file with the USS format option that would fail sending it to the managed node.
-    - Fixed an issue that occurred when unarchiving USS files that would leave temporary files behind on the managed node.
+    - Fixed an issue that when using a local file with the USS format option, the module would fail to send the archive to the managed node.
+    - Fixed an issue that occurred when unarchiving USS files, the module would leave temporary files behind on the managed node.
+
+- ``module_utils``
+
+    - ``job.py`` - Improved exception handling and added a message inside the **content** of the **ddname** when a non-printable
+      character (character that can not be converted to UTF-8) is encountered.
+    - ``data_set.py`` - Fixed an issue that when a volser name less than 6 characters was encountered, the volser name was padded with hyphens to have length 6.
+
 
 Known Issues
 ------------
 
 Several modules have reported UTF-8 decoding errors when interacting with results that contain non-printable UTF-8 characters in the response.
 
-This occurs when a module receives content that does not correspond to a UTF-8 value. These include modules ``zos_job_submit``, ``zos_job_output``,
-``zos_operator_action_query``` but are not limited to this list. This will be addressed in **ibm_zos_core** version 1.10.0-beta.1. Each case is
-unique, some options to work around the error are below.
+- This occurs when a module receives content that does not correspond to a UTF-8 value. These include modules ``zos_job_submit``, ``zos_job_output``,
+  ``zos_operator_action_query``` but are not limited to this list. This has been addressed in this release and corrected with **ZOAU version 1.2.5.6**.
+- If the appropriate level of ZOAU can not be installed, some options are to:
 
-- Specify that the ASA assembler option be enabled to instruct the assembler to use ANSI control characters instead of machine code control characters.
-- Add **ignore_errors:true** to the playbook task so the task error will not fail the playbook.
-- If the error is resulting from a batch job, add **ignore_errors:true** to the task and capture the output into a variable and extract the job ID with
-  a regular expression and then use ``zos_job_output`` to display the DD without the non-printable character such as the DD **JESMSGLG**.
+  - Specify that the ASA assembler option be enabled to instruct the assembler to use ANSI control characters instead of machine code control characters.
+  - Ignore module errors by using  **ignore_errors:true** for a specific.
```

Comment:
maybe missing a "task" at the end of that sentence?
>\- Ignore module errors by using **ignore_errors:true** for a specific.
>\- Ignore module errors by using **ignore_errors:true** for a specific **_playbook task_**.

Reply:
done